### PR TITLE
[april fools] Makes robotics more realistic

### PR DESCRIPTION
**Pull request by NamelessFairy**
## About The Pull Request

SS13 mech construction is extremely unrealistic, you just print parts at the mechfab and stick them together. This PR remedies this and helps put a greater enthesis on realism.
## Why It's Good For The Game

Gives players a sense of pride and accomplishment when assembling a mech.
## Changelog
:cl:
qol: Players familiar with IRL robot construction will now be able to apply their experience to the ingame version
/:cl:

### CODE REVIEW

**Review by Wallemations — APPROVED** [review]
My set didn't ship with a piece

File: code/modules/vehicles/mecha/mecha_parts.dm [hunk 1]
```diff
@@ -8,17 +8,60 @@
...
+/obj/item/mecha_parts/examine(mob/user)
+	. = ..()
+	if(runner)
+		. += span_notice("Use wirecutters to remove the part from the runner.")
 
 /obj/item/mecha_parts/proc/try_attach_part(mob/user, obj/vehicle/sealed/mecha/M, attach_right = FALSE) //For attaching parts to a finished mech
 	if(!user.transferItemToLoc(src, M))
 		to_chat(user, span_warning("\The [src] is stuck to your hand, you cannot put it in \the [M]!"))
 		return FALSE
+	if(runner == TRUE)
```
Comment (LemonInTheDark) [inline code/modules/vehicles/mecha/mecha_parts.dm:33]: why do you do if(runner) in some places and if(runner == true) in others

File: code/modules/vehicles/mecha/mecha_parts.dm [hunk 2]
```diff
@@ -8,17 +8,60 @@
...
 
+/obj/item/mecha_parts/wirecutter_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if(runner == TRUE)
+		if(tool.use_tool(src, user, 2 SECONDS))
+			tool.play_tool_sound(src)
+			runner = FALSE
+			user.visible_message(span_notice("[user] snips [src] from its runner."), span_notice("You snip [src] from its runner."))
+			new /obj/item/runner(get_turf(src), runner_color)
+			underlays.Cut()
```
Comment (LemonInTheDark) [inline code/modules/vehicles/mecha/mecha_parts.dm:47]: please don't

File: code/modules/vehicles/mecha/mecha_parts.dm [hunk 3]
```diff
@@ -8,17 +8,60 @@
...
+	name = "empty runner"
+	icon = 'icons/mob/mecha_runner.dmi'
+	icon_state = "snipped"
+	custom_materials = list(/datum/material/plastic = HALF_SHEET_MATERIAL_AMOUNT)
+	base_pixel_x = -8
+	base_pixel_y = -8
+
+/obj/item/runner/Initialize(mapload, color)
+	. = ..()
+	if(color)
```
Comment (LemonInTheDark) [inline code/modules/vehicles/mecha/mecha_parts.dm:62]: what's it for?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by ArcaneMusic** [comment]
Oh god damnit it's a gundam reference

**Pull request closed without merging**